TACC/Core-Portal
# Task/fp 980 project id command

## Overview: ##

Update project id management command to allow users to determine what the project id should be and to automatically update it.   See jira ticket for more background

## Related Jira tickets: ##

* [FP-980](https://jira.tacc.utexas.edu/browse/FP-980)

## Summary of Changes: ##

- Update project id management command to support automatically updating project id value and ignore high value numbers
- Update README for local dev use of the command
- Add info to README about personalizing the project prefix for developers
- Fix use of token in agave libs

## Testing Steps: ##

Run unit tests and run command in your local environment see idea below on a suggested way to test things:

-  `python3 manage.py projects_id --help`
-  see what the project ids are: `python3 manage.py projects_id`
-  (possibly) kill your local dev database to mimic new dev environment and see what the status of your projects are: ` python3 manage.py projects_id`
-  update them: `python3 manage.py projects_id ----update-using-max-value`
-  set to project id to high value: `python3 manage.py projects_id --update 1000000`
-  create some projects using the portal with these high numbers
-  then revert back to the earlier value:  `python3 manage.py projects_id --update-using-max-value --max-project-id 1000000`

## Notes: ##

- [x] add related PR to `cep1to2` repo to add instructions there (see https://github.com/TACC/cep1to2/pull/13).
